### ISSUE TITLE
It gives an error when I try to convert a jpg to svg

### ISSUE THREAD

**Issue by abadalejandro** [Faster3ck/Converseen#144]
Hello, when I try to convert a jpg image to SVG on my Mac I get an error.
file.jpg converssen: DelegateFailed `'potrace' --svg --output '%o' '%i" @ error/delegate/invokeDelgate/1924
Can you help me?
Thanks for your reply

**Comment by Faster3ck** [comment]
Hello and thank you very much for reporting the bug. The macOS version of Converseen was recently released, so it may be affected by some bugs. I will investigate this problem and release a solution as soon as possible!

**Comment by Faster3ck** [comment]
Yes, unfortunately, as far as I know, it’s not the best tool for this type of conversion. I hope you manage to find the most suitable tool for this kind of conversion! :)
